src/include/planner/operator/simple/logical_import_db.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "planner/operator/simple/logical_simple.h"
+
+namespace kuzu {
+namespace planner {
+
+class LogicalImportDatabase : public LogicalSimple {
+public:
+    LogicalImportDatabase(std::string query, std::shared_ptr<binder::Expression> outputExpression)
+        : LogicalSimple{LogicalOperatorType::IMPORT_DATABASE, std::move(outputExpression)},
+          query{query} {}
+
+    inline std::string getQuery() const { return query; }
+
+    inline std::string getExpressionsForPrinting() const override { return std::string{}; }
+
+    inline std::unique_ptr<LogicalOperator> copy() override {
+        return make_unique<LogicalImportDatabase>(query, outputExpression);
+    }
+
+private:
+    std::string query;
```

Comment:
Not related to this PR but comment / give an example of what this query suppose to be. You can put the comment in `BoundImportDatabase` and point here to the comment.